openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -0,0 +1,74 @@
+from django.urls import reverse
+from django.utils.translation import gettext as _
+
+
+class PersonalizedLearnerScheduleCallToAction:
+    CAPA_SUBMIT_DISABLED = 'capa_submit_disabled'
+    VERTICAL_BANNER = 'vertical_banner'
+
+    def get_ctas(self, xblock, category):
+        """
```

Comment:
Probably doesn't *need* such an extensive reiteration of the class's docstring for this, but doesn't hurt I guess.

Reply:
This isn't a subclass, it's a plugin, so I think it's reasonable to document in both the plugin manager and here.

Reply:
Except documenation is inevitably going to get out of sync. You could maybe point at the main plugin manager class here, if you're worried about folks not being able to find the class with the docs.